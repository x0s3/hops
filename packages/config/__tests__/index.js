/**
 * @jest-environment node
 */

describe('hops-config node export', () => {
  it('should match snapshot', () => {
    expect(require('../index')).toMatchSnapshot();
  });
});
