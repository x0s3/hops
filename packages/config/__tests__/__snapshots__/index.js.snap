// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hops-config node export should match snapshot 1`] = `
Object {
  "_config": Object {
    "name": "hops-lerna-root",
    "rootDir": "/home/zaubernerd/workspace/xing/hops",
    "version": "0.0.0",
  },
  "_env": Object {},
  "_mixins": Object {},
  "_presets": Array [],
  "name": "hops-lerna-root",
  "rootDir": "/home/zaubernerd/workspace/xing/hops",
  "version": "0.0.0",
}
`;
